contrib/sumac/plugins.yml

```diff
@@ -1,2 +1,7 @@
-# TODO this is empty for now. Reach out to contrib maintainers to add items here.
-[]
+- name: aspects
+  src: tutor-contrib-aspects<2.0.0
+  url: https://github.com/openedx/tutor-contrib-aspects
+  author: Brian Mesick <bmesick@axim.org>
+  maintainer: Brian Mesick <bmesick@axim.org>
+  description: |
+    Tutor plugin to install Aspects, the Open edX Learner Analytics System.
```

Comment:
Can I suggest we take this opportunity to update the description? This is totally optional, and if you don't agree then we can just merge this as-is. I suggest you remove the "Tutor plugin" prefix, which is a little redundant, and then add a more extensive description. The "description" field is a multi-line string, the first line of which is used as the plugin "short description". This short description is then used in "tutor plugins search ...". The long description is output by "tutor plugins show ..."

I'm suggesting this because we are working on a UI to display entries from the plugin index, and it will make sense to have a nice description there. Again, totally up to you.